proxy/src/config.rs

```diff
@@ -68,6 +69,12 @@ pub struct Config {
     pub bind_timeout: Duration,
 
     pub pod_namespace: String,
+
+    /// Optional minimum TTL for DNS lookups.
+    pub dns_min_ttl: Option<Duration>,
+
+    /// Optional maximum TTL for DNS lookups.
+    pub dns_max_ttl: Option<Duration>,
```

Comment:
Do you think that the `/etc/resolv.conf` parsing should happen here rather than in `Resolver::from_system_conf` as well, then?

Reply:
> Do you think that the /etc/resolv.conf parsing should happen here rather than in Resolver::from_system_conf as well, then?

OIC, I misunderstood. Never mind. It's fine to keep doing it the original way.